package.json

```diff
@@ -49,27 +49,27 @@
     "types"
   ],
   "devDependencies": {
-    "@microsoft/api-extractor": "~7.20.0",
-    "@pixi-build-tools/api-extractor-lerna-monorepo": "^1.0.3",
-    "@pixi/eslint-config": "^4.0.0",
-    "@pixi/webdoc-template": "^1.5.3",
+    "@microsoft/api-extractor": "~7.33.8",
```

Comment:
Anything with ~ usually means there's a reason we restricted version. I'm not sure what the case is here but probably worth looking through git blame?

Reply:
![image](https://user-images.githubusercontent.com/1616817/214094533-af2d2d64-27d7-4175-8f56-5dae12442b50.png)